client/src/pages/Dialogs/Styles/dialogStyles.js

```diff
@@ -39,33 +39,33 @@ const dialogStyles = makeStyles({
     height: "100%",
   },
   cardDivider: {
-    display: "flex",
+    display: 'flex',
     alignItems: "flex-start",
     padding: 5,
     textAlign: "left",
   },
   cardImageBox: {
-    overflow: "hidden",
+    overflow: 'hidden',
     borderRadius: 2,
-    border: "1px solid #eaeaea",
+    border: '1px solid #eaeaea',
     maxWidth: "25%",
     height: "100%",
   },
   cardImg: {
-    display: "block",
-    width: "100%",
-    height: "auto",
+    display: 'block',
+    width: '100%',
+    height: 'auto',
```

Comment:
It looks like you and @Ricardsss  are using different formatter settings, you need to use the same or the diffs will keep looking messy

Reply:
Yeah, @Ricardsss suggested I use Prettier, I just haven't used it yet. I'll get right on that.